### PR TITLE
Environment variables for default port and bind address

### PR DESCRIPTION
**Pull request by Radical-Egg**
Hi,

There is not an open issue for this but I ran into an issue where I could not change the default port or bind address for snappass. In my case I installed via pip and wanted to bind snappass to 127.0.0.1 as I am planning to use a reverse proxy to reach the app. Is it possible to add the following two environment variables?

BIND_ADDRESS = 127.0.0.1
PORT = 6000

I tested that with no environment variables set, the defaults of 0.0.0.0:5000 are set and working. Given the following variables
```bash
BIND_ADDRESS = 127.0.0.1
PORT = 6000
HOST_OVERIDE = example.com
```
I can reach the app and links are generated with the appropriate URL.

ps. This is my first time submitting a PR, I apologize if I am missing anything.

Thanks all!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -96,6 +96,10 @@ need to change this.
 
 ``HOST_OVERRIDE``: (optional) Used to override the base URL if the app is unaware. Useful when running behind reverse proxies like an identity-aware SSO. Example: ``sub.domain.com``
 
+``BIND_ADDRESS``: (optional) Used to override the default bind address of 0.0.0.0 for flask app Example: ``127.0.0.1``
+
+``PORT``: (optional) Used to override the default port of 5000 Example: ``6000``
+
```
Comment (yurushao) [inline README.rst:102]: Can we prefix them with `SNAPPASS_`?

Comment (Radical-Egg) [reply README.rst:102]: Sure thing, I just made those edits. Thanks for taking a look!

Comment (Radical-Egg) [reply README.rst:102]: Hi @yurushao

I just wanted to check in and see if we would be able to merge this in. 

Thanks again!

**Review by yurushao — APPROVED** [review]

**Comment by yurushao** [comment]
@Radical-Egg Thanks for the PR!

**Pull request merged**